packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -41,9 +41,14 @@ export const CommerceApiContext = React.createContext({} as ApiClients)
 const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
     const {children, clientId, organizationId, shortCode, siteId, proxy} = props
 
+    // DEBUG: copy access token from browser
```

Comment:
Is this a TODO or is there supposed to be a mechanism for setting an access token? I'm not quite sure what this debug statement means.

Reply:
@vcua-mobify that's a temporary code for manually injecting the access token. I'll delete it before merging the PR. Kevin's work would take care of the tokens.